pymatgen/io/ase.py

```diff
@@ -40,7 +40,6 @@
 
 # NOTE: If making notable changes to this class, please ping @Andrew-S-Rosen on GitHub.
 # There are some subtleties in here, particularly related to spins/charges.
-@requires(ase_loaded, "ASE needs to be installed.")
```

Comment:
@Andrew-S-Rosen @shyuep I had to revert the `@requires` decorator added to `AseAtomsAdaptorin` in #3359 because it breaks the `staticmethod` namespace resolution below Python 3.10.